src/Common/src/Interop/Windows/mincore_obsolete/Interop.LocalAlloc_SafeOverlappedFree.cs

```diff
@@ -10,7 +10,9 @@ internal static partial class Interop
 {
     internal static partial class mincore_obsolete
     {
-        [DllImport(Interop.Libraries.Heap, SetLastError = true)]
-        internal static extern SafeOverlappedFree LocalAlloc(int uFlags, UIntPtr sizetdwBytes);
+        internal static int OverlappedSize = IntPtr.Size * 3 + 8;
```

Comment:
This got moved from _Win32.cs. The only thing actually used from that file was the `OVERLAPPED` struct size. Copied the definition from there and replaced in NetworkInformation.

Removed _Win32.cs from Sockets since it wasn't really used.